scripts/sbom_scraper.sh

```diff
@@ -402,67 +477,73 @@ END
 # xmllint complains about a double import of the spdx schema, but we have to import via
 # the wrapper to set the schema location to a local file, as xmllint fails to download
 # them from the internet as they are https
-xmllint "$PATCHED_OUTPUT" --schema "$SCRIPTDIR"/cyclonedx-wrapper.xsd --noout 2>&1 | grep -Fv "Skipping import of schema located at 'http://cyclonedx.org/schema/spdx' for the namespace 'http://cyclonedx.org/schema/spdx'"
-[ "${PIPESTATUS[0]}" -ne 0 ] && exit "${PIPESTATUS[0]}"
+_=$(xmllint "$PATCHED_OUTPUT" --schema "$SCRIPTDIR"/cyclonedx-wrapper.xsd --noout 2>&1 | grep -Fv "Skipping import of schema located at 'http://cyclonedx.org/schema/spdx' for the namespace 'http://cyclonedx.org/schema/spdx'")
```

Comment:
I am wondering do we need this now that the sboms endpoint does proper validation?
Removing the xmllint stuff would make the script simpler and easier to 'sell' to a customer?

Reply:
I think it's still useful especially considering I added `-n` option for when you don't want to upload the SBOM